.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
   - ./gradlew --refresh-dependencies
   - ./gradlew ciWriteBuildNumber
 script:
-  - travis_wait ./gradlew build -x createExe -S
+  - travis_wait ./gradlew build -S
```

Comment:
Correct, `createExe` no longer exists in the build process so in order to fix the ci build I removed it